rust/agents/src/lib.rs

```diff
@@ -40,6 +40,11 @@ pub async fn list() -> Vec<Arc<dyn Agent>> {
         Err(error) => tracing::debug!("While listing Ollama agents: {error}"),
     }
 
+    match agent_anthropic::list().await {
+        Ok(mut agents) => list.append(&mut agents),
+        Err(error) => tracing::debug!("While listing Anthropic agents: {error}"),
+    }
```

Comment:
See the note earlier on in this function - we need a better long term solution perhaps, but for now I think Anthropic should go before ollama.